global/scripts/MyelinMap_BC.sh

```diff
@@ -105,15 +105,15 @@ LeftSphereNewSphere=${LowResFolder}/${Subject}.L.sphere.${LowResMeshString}.surf
 log_File_Must_Exist "${LeftSphereNewSphere}"
 LeftAreaSurfCurrentArea="${NativeT1wFolder}/${Subject}.L.midthickness.native.surf.gii"
 log_File_Must_Exist "${LeftAreaSurfCurrentArea}"
-LeftAreaSurfNewArea=${LowResFolder}/${Subject}.L.midthickness${RegNameInOutputName}.${LowResMeshString}.surf.gii
+LeftAreaSurfNewArea=${LowResT1wFolder}/${Subject}.L.midthickness${RegNameInOutputName}.${LowResMeshString}.surf.gii
```

Comment:
Don't make the same edits on two different branches, it causes issues with merging.  I think we should probably close the other PR that does only this, assuming this PR looks reasonable.